### DISCUSSION TITLE
[Enhancement] using external promerheus for strimzi kafka jmx and other metrics collection instead of using promertheus on same kubernetes cluster

### DISCUSSION THREAD

**Discussion by kvenu1458** [strimzi/strimzi-kafka-operator#6718] [Q&A]
currently as per Strimzi documentation we installed prometheus on same kubernetes cluster where strimzi kafka cluster setup, but using prometheus on same kubernetes cluster will consume space to store all jmx and other metrics, and it might lead to space issue.

So, we want to avoid installing Prometheus on same kubernetes cluster and want to utilze outside Prometheus and collecting jmx metrics and all other metrics

**Comment by scholzj** [comment]
I don't followthe concern about _space issue_. Each component has its own disks which should be (at least for any production like environment) clearly separated. So it is only up to how much disk space you give to your Prometheus cluster and it has nothing to do with whether it runs in the same cluster as Kafka or in a separate one.

But apart from that, Strimzi gives you just some example how you can install and use Prometheus. You are of course free to do it in any way you want. But it is up to you to deploy and configure such setup.
